tests/vector/test_vector_env_wrapper.py

```diff
@@ -8,8 +8,8 @@ def __init__(self, env):
         self.env = env
         self.counter = 0
 
-    def reset_async(self, **kwargs):
```

Comment:
SyncVectorEnv was refactored to not use `reset_async` (because why would it need to do async stuff if it's sync), so this was adapted